bin/arm64_env.sh

```diff
@@ -1,27 +1,4 @@
 #!/bin/bash
 
-export volume=nexus3.edgexfoundry.org:10004/docker-edgex-volume-arm64:master
-export consul=nexus3.edgexfoundry.org:10001/arm64v8/consul:1.3.1
-export configSeed=nexus3.edgexfoundry.org:10004/docker-core-config-seed-go-arm64:master
-export mongo=nexus3.edgexfoundry.org:10004/docker-edgex-mongo-arm64:master
-
-export coreData=nexus3.edgexfoundry.org:10004/docker-core-data-go-arm64:master
-export coreMetadata=nexus3.edgexfoundry.org:10004/docker-core-metadata-go-arm64:master
-export coreCommand=nexus3.edgexfoundry.org:10004/docker-core-command-go-arm64:master
-
-export supportLogging=nexus3.edgexfoundry.org:10004/docker-support-logging-go-arm64:master
-export supportNotifications=nexus3.edgexfoundry.org:10004/docker-support-notifications-go-arm64:master
-export supportScheduler=nexus3.edgexfoundry.org:10004/docker-support-scheduler-go-arm64:master
-export supportRulesengine=nexus3.edgexfoundry.org:10004/docker-support-rulesengine-arm64:master
-
-export deviceVirtual=nexus3.edgexfoundry.org:10004/docker-device-virtual-go-arm64:master
 export appService=nexus3.edgexfoundry.org:10004/docker-app-service-configurable-arm64:latest
```

Comment:
Why isn't this consistent with the rest of the services? This should be pulled from the compose file as well, correct?

Reply:
@lranjbar this is a special case.  There are two application services running for blackbox test.  One is **app-service-rules** and another is **app-service-configurable**.  They are using the same docker image with different configuration because it is **configurable** application service.  The app-service-configurable here is provided for test only, so it doesn't exist in developer-script.
https://github.com/edgexfoundry/developer-scripts/blob/master/releases/nightly-build/compose-files/docker-compose-nexus.yml

You can see the configuration profile here is `blackbox-tests`

```
  app-service-configurable:
    image: ${appService}
    ports:
      - "48095:48095"
    entrypoint: ["/app-service-configurable"]
    command: ["--registry","--confdir=/res", "--profile=blackbox-tests", "--skipVersionCheck=true"]
    container_name: edgex-app-service-configurable
    hostname: app-service-configurable
    environment:
      edgex_registry: consul://edgex-core-consul:8500
      edgex_service: http://app-service-configurable:48095
      edgex_profile: blackbox-tests
      Service_Host: app-service-configurable
      Clients_CoreData_Host: edgex-core-data
      Clients_Logging_Host: edgex-support-logging
      Logging_EnableRemote: "true"
      MessageBus_SubscribeHost_Host: edgex-core-data
      Database_Host: edgex-mongo
      Database_Username: appservice
      Database_Password: password
    networks:
      edgex-network:
        aliases:
          - edgex-app-service-configurable
```

Thus, we need to keep this part in this repository.

Reply:
Then it needs to be updated to use the master tag. There is a discrepancy that shouldn't be there.

Reply:
fixed